build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+function src {
+	mkdir -p "$DIR"/build
+	(cd $DIR/build && cmake ..)
```

Comment:
How come we are launching subshell commands?

Reply:
When stuff has to be run in certain directories, it just makes more sense compared to changing a global state in the code.